.rubocop_todo.yml

```diff
@@ -0,0 +1,110 @@
+# The point is for the user to remove these configuration records
+# one by one as the offenses are removed from the code base.
+# Note that changes in the inspected code, or installation of new
+# versions of RuboCop, may require this file to be generated again.
+
+# Offense count: 1
+# This cop supports safe autocorrection (--autocorrect).
+# Configuration parameters: AllowForAlignment.
+Layout/CommentIndentation:
+  Exclude:
+    - 'lib/puppet/provider/cs_location/pcs.rb'
+
+# Offense count: 4
+# Configuration parameters: IgnoreLiteralBranches, IgnoreConstantBranches.
+Lint/DuplicateBranch:
+  Exclude:
+    - 'lib/puppet/provider/cs_primitive/pcs.rb'
+    - 'spec/spec_helper_acceptance.rb'
+    - 'spec/spec_helper_corosync.rb'
+
+# Offense count: 1
+# This cop supports safe autocorrection (--autocorrect).
+Lint/RedundantCopDisableDirective:
+  Exclude:
+    - 'lib/puppet/provider/cs_colocation/pcs.rb'
+
+# Offense count: 1
+# This cop supports safe autocorrection (--autocorrect).
+Lint/RedundantStringCoercion:
+  Exclude:
+    - 'lib/puppet/provider/cs_rsc_defaults/pcs.rb'
+
+# Offense count: 1
+# Configuration parameters: CheckForMethodsWithNoSideEffects.
+Lint/Void:
+  Exclude:
+    - 'lib/puppet/type/cs_colocation.rb'
+
+# Offense count: 5
+# Configuration parameters: MinNameLength, AllowNamesEndingInNumbers, AllowedNames, ForbiddenNames.
+# AllowedNames: as, at, by, cc, db, id, if, in, io, ip, of, on, os, pp, to
+Naming/MethodParameterName:
+  Exclude:
+    - 'lib/puppet/provider/cs_colocation/pcs.rb'
+    - 'lib/puppet/provider/cs_primitive/crm.rb'
+    - 'lib/puppet/provider/cs_primitive/pcs.rb'
+    - 'lib/puppet/type/cs_primitive.rb'
+    - 'lib/puppet_x/voxpupuli/corosync/provider/cib_helper.rb'
+
+# Offense count: 1
+# Configuration parameters: EnforcedStyle, CheckMethodNames, CheckSymbols, AllowedIdentifiers, AllowedPatterns.
+# SupportedStyles: snake_case, normalcase, non_integer
+# AllowedIdentifiers: capture3, iso8601, rfc1123_date, rfc822, rfc2822, rfc3339, x86_64
+Naming/VariableNumber:
+  Exclude:
+    - 'lib/puppet_x/voxpupuli/corosync/provider/pcs.rb'
+
+# Offense count: 4
+# This cop supports unsafe autocorrection (--autocorrect-all).
+RSpec/BeEq:
+  Exclude:
+    - 'spec/unit/puppet/provider/cs_order_crm_spec.rb'
+    - 'spec/unit/puppet/provider/cs_order_pcs_spec.rb'
+
+# Offense count: 1
+# Configuration parameters: AllowSubject.
+RSpec/MultipleMemoizedHelpers:
+  Max: 10
+
+# Offense count: 4
+RSpec/RepeatedExampleGroupBody:
+  Exclude:
+    - 'spec/acceptance/basic_spec.rb'
+    - 'spec/acceptance/corosync_class_spec.rb'
+
+# Offense count: 2
+# This cop supports safe autocorrection (--autocorrect).
+# Configuration parameters: AllowedMethods, AllowedPatterns.
+# AllowedMethods: ==, equal?, eql?
+Style/ClassEqualityComparison:
+  Exclude:
+    - 'lib/puppet/provider/cs_primitive/crm.rb'
+    - 'lib/puppet/provider/cs_primitive/pcs.rb'
+
+# Offense count: 1
+# This cop supports safe autocorrection (--autocorrect).
```

Comment:
if safe autocorrection is supported, shouldn't `undle exec rake rubocop:autocorrect` fix it automatically?

Reply:
fixed as much rubocop todos as i could while still passing local testing. seems like the class equality comparison, despite claiming to be safely autocorrectible, isnt.